anvil/src/config.rs

```diff
@@ -111,6 +111,8 @@ pub struct NodeConfig {
     pub eth_rpc_url: Option<String>,
     /// pins the block number for the state fork
     pub fork_block_number: Option<u64>,
+    /// specifies the chain id for cache file
```

Comment:
please duplicate offline docs here.

Reply:
I just mentioned offline-start [here](https://github.com/foundry-rs/foundry/compare/3f9ba887901333c50eed57ac4078b3f946fda521..fc12efd78d93ada8530600c2a393418e581fc8bd#diff-3002b4d2e85259099616d3b61841c0113e6bb7de324d701cf2467148cd14bb90R114). You don't want the whole thing, do you?